boris-client/main/boris.hs

```diff
@@ -158,14 +158,16 @@ run c = case c of
               liftIO $ threadDelay 5000000
               pure x
 
-        taillog env l =
-          L.source' env (logDataGroup l) (logDataStream l) $$
-            CL.mapM_ (liftIO . T.putStrLn)
+        taillog (DBLog ls) =
+          T.putStrLn $ renderDBLogs ls
 
       l <- orDie renderBorisHttpClientError waitForLog
-      env <- orDie renderRegionError discoverAWSEnv
       as <- async waitForStatus
-      at <- async $ taillog env l
+      at <- async $ taillog l
+      -- FIXME
+      -- We used to need waitEitherCancel for async fetching
+      -- of logs (CloudWatch) and build-status. Now logs are
+      -- fetched from the db.
```

Comment:
@markhibberd wrapping `taillog` in`async` seems unnecessary now - did it to make the existing `waitEitherCancel` machinery work. How should this change?

Reply:
You don’t need the async/wait it is specific to cloudwatch because cloudwatch doesn’t “know” when a build is done so you have to query the api in parallel with cloudwatch.

Reply:
Yep but what about the async on `waitForStatus` ? If that is still needed then I'm a bit stuck on how to get `waitEitherCancel` to work because it expects two `asyncs`.

Reply:
That is as only to know when to disconnect from cloudwatch, as you are just calling the log you don’t need it.

Reply:
So is `run` doing too much now? Seems like all the code relating to tailing the log is no longer relevant?